pkg/operator/configobservation/cloudprovider/observe_cloudprovider.go

```diff
@@ -0,0 +1,130 @@
+package cloudprovider
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/equality"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/operator/configobservation"
```

Comment:
nit: please move this import down into it sown group

Reply:
fixed